src/groupByKey.ts

```diff
@@ -2,6 +2,22 @@ interface Item {
   [key: string]: any;
 }
 
-export function groupByKey(items, field) {
+interface Result {
+  [key: string]: Item[];
+}
+
+export function groupByKey(items: Item[], field: keyof Item): Result {
   // write code here;
+
+  const result: Item = {};
```

Comment:
```suggestion
  const result: Result = {};
```
This variable should be of a proper type as well

Reply:
I couldn't agree more, thank you ;)